src/odemis/acq/stream/_static.py

```diff
@@ -644,9 +644,10 @@ class StaticSpectrumStream(StaticStream):
     def __init__(self, name, image, *args, **kwargs):
         """
         name (string)
-        image (model.DataArray(Shadow) of shape (CYX) or (C11YX)). The metadata
-        MD_WL_POLYNOMIAL or MD_WL_LIST should be included in order to associate the C to a
-        wavelength.
+        image (model.DataArray(Shadow) of shape (CYX), (C11YX), (CTYX), (CT1YX), (1T1YX)).
+        The metadata MD_WL_POLYNOMIAL or MD_WL_LIST should be included in order to
+        associate the C to a wavelength.
+        The metadata MD_TIME_LIST should be included to associate the T to a timestamp
```

Comment:
Also add info about the .background and .efficiencyCompensation.
.background is a DataArray of shape (CT111), where C & T have the same length as in the data.
.efficiencyCompensation is always DataArray of shape C1111.